borg/cache.py

```diff
@@ -1,6 +1,8 @@
 import configparser
 from .remote import cache_if_remote
+from collections import namedtuple
 import errno
+
 import msgpack
```

Comment:
msgpack is not stdlib, so if you change here, move msgpack down to the 3rd party lib section and keep stdlib in own section.